CHANGELOG.next.toml

```diff
@@ -10,6 +10,15 @@
 # references = ["smithy-rs#920"]
 # meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client | server | all"}
 # author = "rcoh"
+
+[[aws-sdk-rust]]
```

Comment:
This should be `[[smithy-rs]]`. Could you please also add `"target" = "server"` to the meta tags. The change log tool will automatically list changes for the server under a separate heading in the release `CHANGELOG.md`.

Thanks.

Reply:
ah, I missed that. Should have picked up on it when adding the "target" had failed validation.